src/ui/less/components/tabView.less

```diff
@@ -13,13 +13,16 @@
   text-align: center;
   color: @theme-text-color;
   list-style: none;
-  border-top: 1px solid transparent;
+  border: 1px solid black;
```

Comment:
can we go back to transparent here? not sure if this was switched for debugging purposes, looks funky tho